### PR TITLE
US2072 -> not displaying national tag description if no national item…

### PR DESCRIPTION
**Pull request by froberg**
… in hits

### CODE REVIEW

**Comment by froberg** [comment]
@leewadhams can I merge?

**Comment by froberg** [comment]
@mapaux can I merge?

File: src/Web/Sfa.Das.Sas.ApplicationServices/ProviderSearchService.cs [hunk 1]
```diff
@@ -68,28 +68,42 @@ public sealed class ProviderSearchService : IProviderSearchService
         public async Task<ProviderFrameworkSearchResults> SearchFrameworkProviders(int frameworkId, string postCode, Pagination pagination, IEnumerable<string> deliveryModes, bool nationalProviders, bool showAll)
         {
             ProviderFrameworkSearchResults result;
+            //if (!showAll & !nationalProviders)
```
Comment (froberg) [inline src/Web/Sfa.Das.Sas.ApplicationServices/ProviderSearchService.cs]: Sorry missed to delete this.

**Comment by mapaux** [comment]
Yes, you can ;)

**Pull request merged**